hugo/assets/scss/components/cta.scss

```diff
@@ -2,49 +2,74 @@
 @import '../config/sizes';
 @import '../mixins/marginless-child';
 @import '../mixins/screen';
+@import '../mixins/typography';
 
 .cta {
     $self: &;
 
-    background-color: $c-white;
-    border-radius: 5px;
-    color: $c-black;
-    display: block;
-    padding: .75rem;
+    align-items: center;
+    background-color: var(--cta-background, $c-white);
+    color: var(--cta-color, $c-blue);
+    display: flex;
+    flex-direction: column;
+    font-size: 1.25rem;
+    gap: $p-gutter;
+    line-height: 1.75;
+    margin: 0 (-$p-gutter) 2rem;
+    padding: 3rem $p-gutter;
     position: relative;
-
-    &__header {
-        border-bottom: 2px solid $c-black;
-        padding: .75rem;
-    }
+    text-align: center;
 
     &__title {
-        font-size: 1rem;
-        margin: 0;
+        @include style-heading-1;
+
+        color: var(--cta-color);
+        margin: 0
     }
 
     &__content {
         @include marginless-child;
-
-        padding: .75rem .75rem .5rem;
     }
 
-    &--yellow {
-        background-color: $c-yellow;
+    &--flat {
+        --cta-color: #{ $c-blue };
+
+        align-items: normal;
+        border-radius: 5px;
+        gap: 0;
+        font-size: 1rem;
+        line-height: inherit;
```

Comment:
is it needed to set to inherit? I thought that was default behaviour so maybe this line can be removed?

Reply:
The lineheight of the flat differs from the regular CTA. The one from the regular CTA is smaller than the body text. The lineheight of --flat needs to be the same as the body. With inherit it gets the same lineheight of the body.

I can also give it the same value as the body lineheight, but than you have to change it in multiple places if it needs to be adjusted.

Reply:
ah ok, I though inherit took the value of whatever parent with a line-height comes first. Misunderstood that :)